### PR TITLE
Update Addon API min ver

### LINKED ISSUE #12426: many addons have been disabled after updating to snapshot Version: alpha-22782,7e6561a0 U
### Steps to reproduce:
  After updating
  ### Actual behavior:
  addon(s) do not work as well as their settings have been omitted from NVDA preferences.
  ### Expected behavior:
  I can use addon like instantTranslate-4.4.1 or quickDictionary-2.1.2-dev
  ### System configuration
  #### NVDA installed/portable/running from source:
  NVDA installed
  #### NVDA version:
  alpha-22782,7e6561a0 U
  #### Windows version:
  Windows 10 64-bit the last version. (up to date)
  #### Name and version of other software in use when reproducing the issue:
  
  #### Other information about your system:
  Laptop 8GB of RAM, processor: coreI7 64-bit
  ### Other questions
  #### Does the issue still occur after restarting your computer?
  yes it does.
  #### Have you tried any other versions of NVDA? If so, please report their behaviors.
  earlier snapshot versions before around May 14th 2021 worked well.
  #### If add-ons are disabled, is your problem still occurring?
  N/A
  #### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
  N/A

### PR DESCRIPTION
**Pull request by feerrenrut**
2021.1 will have several API breaking changes.
Addons need to be re-tested and have their manifest updated.

This does not need to be merged until 2021.1 branches off for beta. Doing so earlier would cause add-ons to be marked as incompatible for alpha users. However by the time the beta comes, addon authors should have been able to test and provide pre-release versions of their addons.

### CODE REVIEW

**Review by seanbudd — DISMISSED** [review]

**Comment by leonardder** [comment]
Given there are more and more add-ons breaking, I'd like to propose to merge this ASAP. See also #12426

File: source/addonAPIVersion.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 """
 
 CURRENT = (buildVersion.version_year, buildVersion.version_major, buildVersion.version_minor)
-BACK_COMPAT_TO = (2019, 3, 0)
+BACK_COMPAT_TO = (2021, 1, 0)
 """
 As BACK_COMPAT_TO is incremented, the changed / removed parts / or reasoning should be added below.
 EG: (x, y, z): Large changes to speech.py
```
Comment (leonardder) [inline source/addonAPIVersion.py:19]: I think there should be some reasoning below this line about the version updating.

**Comment by feerrenrut** [comment]
We intentionally haven't merged this because:
- Merging this signifies the commitment to the API. 
- Once we merge this, all addon's must update their manifest before they can even test for compatibility. Delaying this process allows authors and adventurous addon users to test and report problems with the upcoming API without a new addon release being made.
- The downside is that users are not warned that add-ons may not work. However remember, this is alpha software, there will be rough edges.

File: source/addonAPIVersion.py (hunk 1 above)
Comment (feerrenrut) [reply source/addonAPIVersion.py:19]: Hmm, yes you are right. Though given the size of the list, it might be better to refer to the changelog.

**Comment by josephsl** [comment]
Hi, I think a representative sample of about two or three major things might be useful to add – I can think of wxPython 4.1.1, say all handler refactor, to name a few. Thanks.

**Review by LeonarddeR — APPROVED** [review]
Looks good to me now.

**Comment by leonardder** [comment]
> We intentionally haven't merged this because:

Makes sense. Having said that, i think this might yet be the time to go with this change:

1. Alpha started with the 2021.2 release cycle, so I assume the API of 2021.1 can be considered stable?
2. It will urge add-on authors once more to fix their add-ons.

**Comment by feerrenrut** [comment]
I believe we are very close to doing this.

>Alpha started with the 2021.2 release cycle, so I assume the API of 2021.1 can be considered stable?

Note quite, we allowed 2021.2 to start so that changes intended specifically for that release could be merged without effecting 2021.1. 

> It will urge add-on authors once more to fix their add-ons.

Yes, that would be good.

**Pull request merged**